pkg/pull/pull.go

```diff
@@ -269,6 +275,24 @@ func Pull(upstreamURI string, pullOptions PullOptions) (string, error) {
 	}
 	log.FinishSpinner()
 
+	newHelmCharts, err := kotsutil.LoadHelmChartsFromPath(fetchOptions.RootDir)
+	if err != nil {
+		log.FinishSpinnerWithError()
+		return "", errors.Wrap(err, "failed to load new helm charts")
+	}
+
+	for _, prevChart := range prevHelmCharts {
+		for _, newChart := range newHelmCharts {
+			if prevChart.Spec.Chart.Name != newChart.Spec.Chart.Name {
+				continue
+			}
+			if prevChart.Spec.UseCLIInstall != newChart.Spec.UseCLIInstall {
+				log.FinishSpinnerWithError()
+				return "", errors.Errorf("deployment method for chart %s has changed", newChart.Spec.Chart.Name)
```

Comment:
i thought you just cant change it from UseCLIInstall to ! UseCLIInstall

Reply:
Toggling is untested.  Future feature?

Reply:
So this works on new installs only?

Reply:
Yes, that was the intention.